### PR TITLE
Start a changelog

### PR DESCRIPTION
**Pull request by nedbat**

### CODE REVIEW

**Review by Mariatta — APPROVED** [review]

**Comment by Mariatta** [comment]
Could you link to this from the readme.md?

**Comment by Mariatta** [comment]
Thank you!

**Comment by nedbat** [comment]
Added a link from the readme.

File: README.md [hunk 1]
```diff
@@ -6,7 +6,9 @@ See [PEP 732](https://peps.python.org/pep-0732/).
 
 ## Updates
 
-These updates provide high level information about the Editorial Board's activities.
+We have a "changelog" of decisions: [CHANGELOG.md](CHANGELOG.md).
```
Comment (Mariatta) [inline README.md:9]: Hmm I think the `Updates` section should list our monthly notes.
I would suggest just adding a new header/section for Changelog

eg 

```
## Changelog

We have a "changelog" of decisions: [CHANGELOG.md](CHANGELOG.md). 
```

Comment (nedbat) [reply README.md]: done

**Comment by Mariatta** [comment]
Thanks!

**Review by ezio-melotti — APPROVED** [review]

**Review by willingc — APPROVED** [review]

**Pull request merged**